roachcase/_infrastructure/dynamodb_repositories.py

```diff
@@ -41,6 +41,13 @@ def create_table(self, specs: Dict[str, Any]) -> None:
             waiter = self.__client.get_waiter("table_exists")
             waiter.wait(TableName=table_name)
 
+    def remove(self, table_name: str, item: Dict[str, Any]) -> None:
```

Comment:
This is duplicated of `delete`. I don't mind which one we use, but we should have only one of them.